tests/rptest/clients/kubectl.py

```diff
@@ -119,7 +119,13 @@ def _scp_cmd(self, src, dest):
     def _install(self):
         '''Installs kubectl on a remote target host'''
         if not self._kubectl_installed and self._remote_uri is not None:
-            self._ssh_cmd(['./breakglass-tools.sh'])
+            breakglass_cmd = ['./breakglass-tools.sh']
+            if self._provider == 'azure':
+                # for azure, we manually override the path here to ensure
+                # that azure-cli installed as a snap gets found (workaround)
+                p = ['env', 'PATH=/usr/local/bin:/usr/bin:/bin:/snap/bin']
```

Comment:
should this append `PATH` at the end? just to ensure that other binaries that might be implicitly invoked (if any) can be found

Reply:
Short answer: no need!

Longer answer: I don't know if environment variable expansion works with the options we use, so I assumed it wouldn't and grabbed the default full `$PATH` (including `/snap/bin`) from an interactive shell, and then I removed `/usr/games` and `/usr/local/games` after confirming that both were completely empty on the agent VM's ubuntu install. So adding `$PATH` will only result in duplicating existing paths and adding empty directories to the search index.

Reply:
Is there an open bug/issue for cloud side to make azure cli work w/o this hack?

Reply:
> Is there an open bug/issue for cloud side to make azure cli work w/o this hack?

Yes, it's filed with them as https://github.com/redpanda-data/cloudv2/issues/16371